metadata-ingestion/src/datahub/emitter/rest_emitter.py

```diff
@@ -503,31 +485,36 @@ def _emit_openapi_mcps(
         trace_timeout: Optional[timedelta] = timedelta(seconds=3600),
     ) -> int:
         """
-        1. Grouping MCPs by their entity URL
+        1. Grouping MCPs by their entity URL and HTTP method
```

Comment:
A minor suggestion to perfectly match the code.

```suggestion
        1. Grouping MCPs by their HTTP method and entity URL
```